packages/common/source/models/MembershipModel.ts

```diff
@@ -0,0 +1,43 @@
+import { Schema, model } from "mongoose";
+
+import { Membership } from "../types";
+import { membershipStatuses, membershipTypes } from "../utils/constants";
+
+const membershipSchema = new Schema(
+    {
+        // An identifier that points to the User whose membership is being defined by the current document.
```

Comment:
How many times should I tell you to not use `//` comments? Replace all these comments with `/* ... */`. Also, document the interfaces. Use doc comments there. Learn about doc comments before you continue.